packages/flame_console/example/pubspec.yaml

```
@@ -0,0 +1,24 @@
+name: example
```

```suggestion
name: flame_console_example
```
If you name it `example` it will cause clashes.